src/routes/console/project-[project]/messaging/providers/store.ts

```diff
@@ -0,0 +1,151 @@
+import { writable } from 'svelte/store';
+import type { Column } from '$lib/components/viewSelector.svelte';
+
+export let showCreate = writable(false);
+
+export const columns = writable<Column[]>([
+    { id: '$id', title: 'Provider ID', show: true },
+    { id: 'name', title: 'Name', show: true },
+    { id: 'provider', title: 'Provider', show: true },
+    { id: 'channel', title: 'Channel', show: true },
+    { id: 'status', title: 'Status', show: true }
+]);
+
+export const providers = {
```

Comment:
@TorstenDittmann, what do you think about having this JSON for the wizard for all providers. I wanted to try and minimize the number of places you'd need to go if we add a messaging provider in the future.

Reply:
Torsten: for now, let's continue with this.